### ISSUE TITLE
Javascript error when trying to edit autoresponse rule for system mailbox

### ISSUE THREAD

**Issue by ianfp** [orocrm/crm#151]
Steps to reproduce:
1) Add a new system mailbox with IMAP enabled
2) Save the new mailbox
3) Edit the mailbox and add an autoresponse rule
4) Save the autoresponse rule
5) Try to edit the new autoreponse rule again by clicking the edit icon in the grid

I get this error in my Javascript console:
```
TypeError: e is undefined
t<.openWidget()
 oro.min.js:1084
o<.run()
 oro.min.js:1102
t<.onClick()
 oro.min.js:1088
Q.event.dispatch()
 oro.min.js:1063
Q.event.add/g.handle()
 oro.min.js:1063
 oro.min.js
```

I'm using OroCRM 1.9.0, Firefox 44.0.2, Ubuntu 14.04.3.

**Comment by Grygir** [comment]
Thank you for the report. It will be fixed in 1.9.1

**Comment by amuzalevskiy** [comment]
Fix is merged and will be available in platform v1.9.1

Thank you
